src/duckling/game/drawable/container-drawable.component.ts

```diff
@@ -57,6 +59,17 @@ export class ContainerDrawableComponent {
     @Input() containerDrawable : ContainerDrawable;
     @Output() drawableChanged = new EventEmitter<ContainerDrawable>();
 
+    onChildDrawableCloned(newDrawables : Drawable[]) {
```

Comment:
If this is the type of work we need to do when something is cloned, we should probably change the API provided by the accordion.